### PR TITLE
fix parse_report

### PR DESCRIPTION
**Pull request by nickpell**
```strop``` is removed in py3

### CODE REVIEW

File: custom/ilsgateway/tanzania/handlers/soh.py [hunk 1]
```diff
@@ -39,19 +38,26 @@ def parse_report(val):
...
+        from strop import maketrans
+
+    if six.PY3:
+        val = six.text_type(val)
+    else:
+        val = six.text_type(val).encode('utf-8')
 
     return [
-        (x[0], int(x[1].translate(maketrans("lLO", "110"))))
+        (x[0], int(x[1].replace(' ', '').translate(maketrans("lLO", "110"))))
```
Comment (millerdev) [inline custom/ilsgateway/tanzania/handlers/soh.py]: This appears to be new functionality, not a py2 -> py3 compatibility change. Can you explain the reason for it or what I'm missing?

**Review by millerdev — APPROVED** [review]

File: custom/ilsgateway/tanzania/handlers/soh.py [hunk 2]
```diff
@@ -39,19 +38,26 @@ def parse_report(val):
...
-        return six.text_type(s).encode('utf-8')
+    if six.PY3:
+        maketrans = str.maketrans
+    else:
+        from strop import maketrans
+
+    if six.PY3:
+        val = six.text_type(val)
+    else:
+        val = six.text_type(val).encode('utf-8')
```
[the remark this answers is not in the document]
Comment (nickpell) [reply custom/ilsgateway/tanzania/handlers/soh.py]: updated

File: custom/ilsgateway/tanzania/handlers/soh.py (hunk 1 above)
Comment (nickpell) [reply custom/ilsgateway/tanzania/handlers/soh.py]: This was a breaking change in python 2, so I instead decided to fork the function.  The py3 tests pass, so the python 3 functionality should be preserved.

**Comment by nickpell** [comment]
✅

**Review by millerdev — APPROVED** [review]

**Pull request merged**